Source/Krypton Components/Krypton.Toolkit/View Base/ViewManager.cs

```diff
@@ -202,7 +202,7 @@ public bool OutputDebug
         /// <param name="renderer">Renderer provider.</param>
         /// <exception cref="ArgumentNullException"></exception>
         /// <returns>True if it contains transparent painting.</returns>
-        public bool EvalTransparentPaint([DisallowNull] IRenderer renderer)
```

Comment:
Leave the `[DissallowNull]`'s in there, so that it can be visually checked that this code has been checked for nullables.